zcash-issue-dag.py

```diff
@@ -43,10 +43,9 @@
     151763639: ('zcash', 'zcash-android-wallet-sdk'),
     159714694: ('zcash', 'lightwalletd'),
     185480114: ('zcash', 'ZcashLightClientKit'),
-    223814143: ('zcash', 'zcash-android-wallet'),
-    225922879: ('zcash', 'zcash-ios-wallet'),
     387551125: ('zcash', 'secant-ios-wallet'),
     390808594: ('zcash', 'secant-android-wallet'),
+    270825987: ('zcash-hackworks', 'MnemonicSwift'),
```

Comment:
I did not verify that the GitHub project ID here is correct, but I presume you tested this.